### ISSUE TITLE
Error visiting / after creating a new app

### ISSUE THREAD

**Issue by skillmatic-co** [kevinhughes27/shopify-sinatra-app#41]
I'm getting error:

```
ArgumentError - wrong number of arguments (given 2, expected 0; required keywords: domain, token, api_version):
```

At line 14 of app.rb, which is this line:

```
shopify_session do |shop_name|
```

Did something change with the API where more details are needed in creating this session?

Full stack trace:

```
2019-04-25 21:49:21 - ArgumentError - wrong number of arguments (given 2, expected 0; required keywords: domain, token, api_version):
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/shopify_api-7.0.1/lib/shopify_api/session.rb:87:in `initialize'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/shopify-sinatra-app-0.7.0/lib/sinatra/shopify-sinatra-app.rb:86:in `new'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/shopify-sinatra-app-0.7.0/lib/sinatra/shopify-sinatra-app.rb:86:in `activate_shopify_api'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/shopify-sinatra-app-0.7.0/lib/sinatra/shopify-sinatra-app.rb:41:in `shopify_session'
	/Users/foo/www/sites/test-sinatra-app/src/app.rb:14:in `block in <class:SinatraApp>'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:1635:in `call'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:1635:in `block in compile!'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:992:in `block (3 levels) in route!'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:1011:in `route_eval'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:992:in `block (2 levels) in route!'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:1040:in `block in process_route'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:1038:in `catch'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:1038:in `process_route'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:990:in `block in route!'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:989:in `each'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:989:in `route!'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:1097:in `block in dispatch!'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:1076:in `block in invoke'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:1076:in `catch'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:1076:in `invoke'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:1094:in `dispatch!'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:924:in `block in call!'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:1076:in `block in invoke'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:1076:in `catch'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:1076:in `invoke'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:924:in `call!'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:913:in `call'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/omniauth-1.9.0/lib/omniauth/strategy.rb:192:in `call!'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/omniauth-1.9.0/lib/omniauth/strategy.rb:169:in `call'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/omniauth-1.9.0/lib/omniauth/builder.rb:64:in `call'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/rack-2.0.7/lib/rack/session/abstract/id.rb:232:in `context'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/rack-2.0.7/lib/rack/session/abstract/id.rb:226:in `call'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/rack-2.0.7/lib/rack/method_override.rb:22:in `call'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/rack-flash3-1.0.5/lib/rack/flash.rb:124:in `call'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/rack-protection-2.0.5/lib/rack/protection/xss_header.rb:18:in `call'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/rack-protection-2.0.5/lib/rack/protection/base.rb:50:in `call'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/rack-protection-2.0.5/lib/rack/protection/base.rb:50:in `call'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/rack-protection-2.0.5/lib/rack/protection/path_traversal.rb:16:in `call'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/rack-protection-2.0.5/lib/rack/protection/json_csrf.rb:26:in `call'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/rack-protection-2.0.5/lib/rack/protection/base.rb:50:in `call'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/rack-protection-2.0.5/lib/rack/protection/base.rb:50:in `call'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/rack-2.0.7/lib/rack/session/abstract/id.rb:232:in `context'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/rack-2.0.7/lib/rack/session/abstract/id.rb:226:in `call'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/rack-2.0.7/lib/rack/null_logger.rb:9:in `call'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/rack-2.0.7/lib/rack/head.rb:12:in `call'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/show_exceptions.rb:22:in `call'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:194:in `call'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:1957:in `call'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:1502:in `block in call'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:1729:in `synchronize'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/sinatra-2.0.5/lib/sinatra/base.rb:1502:in `call'
	/Users/foo/.rvm/gems/ruby-2.5.3/gems/rack-2.0.7/lib/rack/handler/webrick.rb:86:in `service'
	/Users/foo/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/webrick/httpserver.rb:140:in `service'
	/Users/foo/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/webrick/httpserver.rb:96:in `run'
	/Users/foo/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/webrick/server.rb:307:in `block in start_thread'
```

**Comment by skillmatic-co** [comment]
It seems so: https://github.com/Shopify/shopify_api/blob/master/README.md#-breaking-change-notice-for-version-700-

**Comment by skillmatic-co** [comment]
Created PR: https://github.com/kevinhughes27/shopify-sinatra-app/pull/42

**Comment by kevinhughes27** [comment]
closed by #42
